spki

```diff
@@ -208,81 +224,99 @@ create () {
 	PASS=$(get-password 'private key')
 	exec 3<<<"$PASS"
 	if [[ -z "$CERT_ENCRYPTION" ]]; then
-		openssl genrsa -out $KEY -passout fd:3 2048 |& indent
+		openssl genrsa -out "$KEY" -passout fd:3 2048 |& indent
 	else
-		openssl genrsa -"$CERT_ENCRYPTION" -out $KEY -passout fd:3 2048 |& indent
-	fi && \
-	chmod 400 $KEY && \
-	echoc 'Creating CSR' green
+		openssl genrsa -"$CERT_ENCRYPTION" -out "$KEY" -passout fd:3 2048 |& indent
+	fi &&
+		chmod 400 "$KEY" &&
+		echoc 'Creating CSR' green
 	exec 3<<<"$PASS"
 	if [[ "$3" == "-SAN" ]]; then
-		openssl req -verbose -reqexts SAN -config <(cat $INTRMDT_CONF \
-        	<(printf "\n[SAN]\nsubjectAltName=$4")) \
-			-key $KEY \
-			-new -sha256 -out $CSR -passin fd:3
+		openssl req -verbose -reqexts SAN -config <(cat "$INTRMDT_CONF" \
+			<(printf "\n[SAN]\nsubjectAltName=%s" "$4")) \
+			-key "$KEY" \
+			-new -sha256 -out "$CSR" -passin fd:3
 	else
-		openssl req -config $INTRMDT_CONF \
-			-key $KEY \
-			-new -sha256 -out $CSR -passin fd:3
-	fi && \
-	sign $1 $CSR $CERT || { echoc 'Could not create certificate' red 1>&2; return 1; }
+		openssl req -config "$INTRMDT_CONF" \
+			-key "$KEY" \
+			-new -sha256 -out "$CSR" -passin fd:3
+	fi &&
+		sign "$1" "$CSR" "$CERT" || {
+		echoc 'Could not create certificate' red 1>&2
+		return 1
+	}
 	echo
 	echoc "Successfully created the following files" greenbold
 	echo "Key: $KEY" | indent
 	echo "Certificate: $CERT" | indent
 }
 
-create-intermediate () {
+create-intermediate() {
 	# https://jamielinux.com/docs/openssl-certificate-authority/create-the-intermediate-pair.html
 	echoc 'Creating intermediate CA private key' green
 	INTRMDT_PASS=$(get-password 'intermediate CA private key')
 	exec 3<<<"$INTRMDT_PASS"
-	openssl genrsa -aes256 -out $INTRMDT_KEY -passout "fd:3" 4096 |& indent && \
-	chmod 400 $INTRMDT_KEY || { echoc 'Could not create intermediate CA private key' red 1>&2; return 1; }
+	openssl genrsa -aes256 -out "$INTRMDT_KEY" -passout "fd:3" 4096 |& indent &&
+		chmod 400 "$INTRMDT_KEY" || {
+		echoc 'Could not create intermediate CA private key' red 1>&2
+		return 1
+	}
 
 	exec 3<<<"$INTRMDT_PASS"
-	echoc 'Creating intermediate CA CSR' green && \
-	openssl req -verbose -config $INTRMDT_CONF -new -sha256 \
-		-key $INTRMDT_KEY \
-		-out $INTRMDT_CSR -passin "fd:3" || { echoc 'Could not create intermediate CA CSR' red 1>&2; return 1; }
+	echoc 'Creating intermediate CA CSR' green &&
+		openssl req -verbose -config "$INTRMDT_CONF" -new -sha256 \
+			-key "$INTRMDT_KEY" \
+			-out "$INTRMDT_CSR" -passin "fd:3" || {
+		echoc 'Could not create intermediate CA CSR' red 1>&2
+		return 1
+	}
 
-	if [[ -n "$ROOT_PASS" ]]; then
+	if [[ -n $ROOT_PASS ]]; then
```

Comment:
should be quoted too

Reply:
see https://github.com/kaysond/spki/pull/9#discussion_r353624971